app/Home.tsx

```diff
@@ -44,7 +44,7 @@ export default function Home() {
             </Link>
           </div>
           <Link href="https://github.com/mohyware/clip.js" target="_blank"
-            className="rounded-full bg-black border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
+            className="rounded-full bg-black border border-solid border-transparent transition-colors flex items-center justify-center text-white gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
```

Comment:
```suggestion
            className="rounded-full bg-black border border-solid border-transparent transition-colors flex items-center justify-center text-white gap-2 hover:bg-[#383838] dark:hover:bg-[#383838] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
```
The old hover color is really better than this all-white one.

The old
![image](https://github.com/user-attachments/assets/ec04905c-de72-426e-92b9-27b16926392a)

The current
![image](https://github.com/user-attachments/assets/c12e6aae-38c6-42cc-ad0c-f82ce9474b29)